### DISCUSSION TITLE
Apiato api testing with postman

### DISCUSSION THREAD

**Discussion by Bamba-s** [apiato/apiato#713] [General]
Hello
Please, I need help.
I am beginner to apiato, I followed these instructions: "https://apiato.io/docs/9.x/miscellaneous/postman/#apiato-postman-api--environment" to test my API on postman, but I have a problem authenticating as an admin. Here is what postman shows me: "Error: getaddrinfo ENOTFOUND api.apiato.test"

**Comment by yalsicor** [comment]
How did you setup your local dev environment? How are you serving the application locally?
https://apiato.io/docs/9.x/getting-started/installation#Development-Environment

Have you setup your local hosts file? (/etc/hosts)
